test/controllers/surveys_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class SurveysControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @survey = surveys(:one)
```

Comment:
try to use `factory` gem, it's better

Reply:
Seem like this one was auto generated by `rails generate scaffold`, I did not plan to test it at the moment. Should I remove it? Or just leave it there and update with proper test later